openmetadata-ui/src/main/resources/ui/src/WebAnalytics/WebAnalyticsProvider.test.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { act, render, screen } from '@testing-library/react';
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import WebAnalyticsProvider from './WebAnalyticsProvider';
+
+describe('Test WebAnalytics Component', () => {
+  it('Should render the child component', async () => {
+    await act(async () => {
+      render(
+        <WebAnalyticsProvider>
+          <div data-testid="mock-children">Mock Children</div>
+        </WebAnalyticsProvider>,
+        {
+          wrapper: MemoryRouter,
+        }
+      );
+    });
+
+    const children = await screen.findByTestId('mock-children');
```

Comment:
This test isn't making any sense. What we are trying to test here?

Reply:
Just the basics if it's rendering the children or not.